pkg/reconciler/knativeserving/controller.go

```diff
@@ -55,6 +55,7 @@ func NewController(
 	c := &Reconciler{
 		Base:                 rbase.NewBase(ctx, controllerAgentName, cmw),
 		knativeServingLister: knativeServingInformer.Lister(),
+		servings:             map[string]bool{},
```

Comment:
Do you want to use a [`sets.String`](https://godoc.org/k8s.io/apimachinery/pkg/util/sets#String) here? It's not a lot more space to store the bool, but then you have to wonder if there's a meaning to `true` and `false`.

Reply:
Good call. Done.